integration/images/2.2/Dockerfile

```diff
@@ -0,0 +1,64 @@
+FROM ruby:2.2.10
+
+# Add Jessie repos
+# Fixes https://superuser.com/questions/1423486/issue-with-fetching-http-deb-debian-org-debian-dists-jessie-updates-inrelease
+RUN printf "deb http://archive.debian.org/debian/ jessie main\ndeb-src http://archive.debian.org/debian/ jessie main\ndeb http://security.debian.org jessie/updates main\ndeb-src http://security.debian.org jessie/updates main" > /etc/apt/sources.list
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Install prerequisites
+RUN set -ex; \
+        apt-get update; \
+        apt-get install -y --force-yes --no-install-recommends \
+            curl wget tar gzip gnupg apt-transport-https ca-certificates tzdata locales; \
+        rm -rf /var/lib/apt/lists/*;
+
+# Install NodeJS
+RUN set -ex; \
+        apt-get update; \
+        apt-get install -y --force-yes --no-install-recommends nodejs; \
+        rm -rf /var/lib/apt/lists/*;
+
+# Install Yarn
+RUN set -ex; \
+        curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -; \
+        echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list;
+
+RUN set -ex; \
+        apt-get update; \
+        apt-get install -y --force-yes --no-install-recommends yarn; \
+        rm -rf /var/lib/apt/lists/*;
+
+# Install databases
+RUN set -ex; \
+        apt-get update; \
+        apt-get install -y --force-yes --no-install-recommends \
+            postgresql-client sqlite3; \
+        rm -rf /var/lib/apt/lists/*;
+
+# Install dev tools
+RUN set -ex; \
+        apt-get update; \
+        mkdir -p /usr/share/man/man1; \
+        apt-get install -y --force-yes --no-install-recommends \
+            sudo git openssh-client rsync vim \
+            net-tools netcat parallel unzip zip bzip2; \
+        rm -rf /var/lib/apt/lists/*;
```

Comment:
Minor: Have you considered combining all of the installation steps? That would allow us to avoid running `apt-get update` a few extra times. (Thus making image builds faster with little downsides).

This would also apply to the other `Dockerfile`s.

Reply:
Yeah, would be okay. More work for a minor gain though, and I think there's something nice about having blocks? (In case we want to remove/add steps in the future?) If the gain is big enough, I'd be fine with this.

Reply:
Completely agreed on the the blocks as well; I was thinking that the approach in [`integration/images/wrk/Dockerfile`](https://github.com/DataDog/dd-trace-rb/pull/1305/files#diff-8beb1ed24e2cd6592daa4531f7356ed95133ebc692e8cdfbd645b704567bac44) seems pretty reasonable to get the best of both worlds :)

Reply:
I see: visually separate but keep logically in one block. I think this could be fine.

Reply:
Implemented this.